mapboxcli/scripts/geocoding.py

```diff
@@ -98,7 +101,7 @@ def geocoding(ctx, query, forward, include_headers, lat, lon,
         for q in iter_query(query):
             try:
                 resp = geocoder.forward(
-                    q, types=place_type, lat=lat, lon=lon, country=country)
+                    q, types=place_type, lat=lat, lon=lon, country=country, bbox=bbox)
```

Comment:
Since the argument will be coming in as text, we'll need to parse it into a list as expected by the sdk.

If we're going to assume that the `bbox` is a string containing a 4-element JSON array (`[minX,minY,maxX,maxY]`) we could do something like:

```
try:
    bbox = json.loads(bbox)
except ValueError:
    raise click.BadParameter("--bbox must be a JSON array")
```

I'm always wavering on the decision about how to represent bounds - some of our tools use `--bounds w s e n` (4 args) versus a single array `--bounds [w,s,e,n]`. Do you have a preference in this context @sgillies?